src/SensioLabs/Melody/Tests/Handler/GistHandlerTest.php

```diff
@@ -6,7 +6,7 @@
 
 class GistHandlerTest extends \PHPUnit_Framework_TestCase
 {
-    const SINGLE_URL = 'https://gist.github.com/csarrazi/7494d27255d0561157b8';
```

Comment:
Regardless, instead of doing that, we should update the test so it actually uses a mock and not a real http call.